### PR TITLE
Remove logo and change color

### LINKED ISSUE #20: Maintenance-Page: remove logo and colors
1) The generated maintenance page has a transparent 4teamwork **logo**, white text on white background, the text is not readable.
I'd prefer not having a logo at all by default, so that the maintenance-page can be used for any project without the need to replace the logo. The easiest way is to just not have a logo :wink: 

2) The page currently has a **green border**. The green may not be appropriate for e.g. a red-ish page.
In order to not have this problem I'd propose to not use a color and just use a greyscale page.

This is not urgent.
Maybe @bierik can make a nice looking maintenance page sometime? :wink:

### PR DESCRIPTION
**Pull request by raphael-s**
Removes the logo from the maintenance page and sets a neutral color.

bildschirmfoto 2016-05-19 um 10 27 44

closes #20

### CODE REVIEW

File: ftw/bob/web/template/+package.fullname+/maintenance/index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+w<!doctype html>
```
Comment (jone) [inline ftw/bob/web/template/+package.fullname+/maintenance/index.html]: 🙊

**Comment by jone** [comment]
👍

**Comment by raphael-s** [comment]
Hoppla 🙈 

Ist gefixed @jone

**Comment by jone** [comment]
@raphael-s can you also remove the logo-image from the maintenance folder? It is no longer necessary.

**Comment by raphael-s** [comment]
Done @jone

**Pull request merged**